src/components/v2/shared/V2CurrencyAmount.tsx

```diff
@@ -14,7 +14,7 @@ export default function V2CurrencyAmount({
 }) {
   if (!amount) return null
 
-  if (currency?.eq(V2_CURRENCY_ETH)) {
+  if (currency?.eq(0) || currency?.eq(V2_CURRENCY_ETH)) {
```

Comment:
I think we should just leave this. `distributionLimitCurrency` is defaulting to 0 in various places. I see no reason to leave this here as a safeguard. Sick of continuing to see non-rendering currency amounts coming back

Reply:
As I understand it, `0` means no value was set. It doesn't mean it's an ETH amount right?